src/main/java/org/traccar/helper/Checksum.java

```diff
@@ -200,4 +200,23 @@ public static long luhn(long imei) {
         return (10 - (checksum % 10)) % 10;
     }
 
+    public static int udp(ByteBuffer data) {
```

Comment:
I think this checksum is also used for TCP. Maybe call it `ip`?

Reply:
I couldn't find other examples of this checksum. The "Internet Checksum" from RFC 791 / 1071 appears to be slightly different. Renamed ip().